Client/Client.Test/TestClient.hpp

```diff
@@ -0,0 +1,189 @@
+#pragma once
+
+#include "Network/Connection.hpp"
+#include <Models/Primitives.hpp>
+#include <FileLogger.hpp>
+
+#include "TestServerInfo.hpp"
+
+namespace TestObject
```

Comment:
And why we need this objects here? Maybe atleast change namespace for undesdanding reason of existing this?